### PR TITLE
add last updated at times

### PR DESCRIPTION
**Pull request by willyb321**
in the result it will show like this
screen shot 2017-07-22 at 20 44 30

### CODE REVIEW

**Comment by SayakMukhopadhyay** [comment]
Good call. Update coming soon<sup>TM</sup>

File: src/modules/discord/commands/factionStatus.ts [hunk 1]
```diff
@@ -110,11 +110,16 @@ export class FactionStatus {
                                                     let influence = responseObject[0].history[0].influence;
                                                     let pendingStatesArray = responseObject[0].history[0].pending_states;
                                                     let recoveringStatesArray = responseObject[0].history[0].recovering_states;
-
                                                     let factionDetail = "";
+                                                    let updatedAt = new Date(responseObject[0].updated_at);
+                                                    let currentTime = new Date();
+                                                    if (currentTime < updatedAt) {
```
Comment (SayakMukhopadhyay) [inline src/modules/discord/commands/factionStatus.ts]: What is this check for? Current time is always going to be lesser than updated one, right?

Comment (willyb321) [inline src/modules/discord/commands/factionStatus.ts]: handles cases where the times are on the opposite side of midnight e.g. when you want to get the difference between 9:00 PM and 5:00 AM

Comment (willyb321) [inline src/modules/discord/commands/factionStatus.ts]: dates can be weird

**Pull request merged**